cypress/support/commands/application.js

```diff
@@ -0,0 +1,35 @@
+Cypress.Commands.add('testArticleServiceWorker200s', service => {
```

Comment:
These commands are far too specific IMO. Commands should share reusable code around our tests. These can only be used in 1 application.